ac/ac-induction/src/config.js

```diff
@@ -22,11 +22,11 @@ export default {
       }
     },
     imgTrue: {
-      title: 'correct image uri',
+      title: 'Correct image URL',
```

Comment:
Right now when I paste a large dataURI in a field of the configuration form. It calls `onChange` and takes time to load. During that time I lose control of the app. Uploading files usually is done in the background to not freeze the interface for the user. You need to handle that issue.

For this you should see how to make the Configuration `onChange` asynchronous.